mods/ra/maps/red-zone/rules.yaml

```diff
@@ -0,0 +1,490 @@
+		RequiresCondition: hard
+		Modifier: 120
+	ExternalCondition@EASY:
+		Condition: easy
+	ExternalCondition@HARD:
+		Condition: hard
+	RenderSprites:
+		Scale: 1.5
+
+HIND.Husk:
+	RenderSprites:
+		Scale: 1.3
+
+APC.EntryUnit:
+	Inherits: APC
+	-Selectable:
+	-Buildable:
+	Interactable:
+	RejectsOrders:
+	Cargo:
+		Types: Infantry
+		MaxWeight: 1
+		PipCount: 1
+	RenderSprites:
+		Image: apc
+
+LST.IN:
+	Inherits: LST
+	-Buildable:
+	RejectsOrders:
+	RenderSprites:
+		Image: LST
+	Cargo:
+		Types: disabled
+
+DOME:
+	RenderSprites:
+		Scale: 1.1
+	Health:
+		HP: 300000
+	Tooltip:
+		Name: Communications Center
+		GenericVisibility: None
+	Power:
+		Amount: 600
+
+DTRK:
+	RejectsOrders:
+	Health:
+		HP: 300000
+	Armor:
+		Type: heavy
+	Mobile:
+		Speed: 100
+
+TRUK:
+	Huntable:
+		RequiresCondition: loaded
+	SpawnActorOnDeath:
+		Actor: healcrate
+	Mobile:
+		RequiresCondition: !being-captured && loaded
+	Cargo:
+		Types: Pilot
+		MaxWeight: 1
+		PipCount: 1
+		InitialUnits: thf
+		RequiresCondition: loaded
+
+JEEP:
+	Inherits@AUTOTARGET: ^AutoTargetAllAssaultMove
+	Huntable:
+		RequiresCondition: loaded
+	Mobile:
+		RequiresCondition: !notmobile && !being-captured && loaded
+	Armament@AA:
+		Weapon: M60mgAA
+		MuzzleSequence: muzzle
+		LocalOffset: 128,0,43
+	Cargo:
+		Types: Infantry, Pilot, Co-Pilot
+		InitialUnits: thf
+		LoadedCondition: loaded
+	AttackTurreted:
+		RequiresCondition: loaded
+	Turreted:
+		RequiresCondition: loaded
+	Targetable@IGNORE:
+		TargetTypes: NoAutoTarget
+		RequiresCondition: !loaded
+
+2TNK:
+	Inherits@AUTOTARGET: ^AutoTargetAllAssaultMove
+	AutoTarget:
+		AllowMovement: False
+	Huntable:
+		RequiresCondition: loaded
+	Mobile:
+		RequiresCondition: !being-captured && loaded
+	Cargo:
+		Types: Pilot
+		MaxWeight: 1
+		PipCount: 1
+		InitialUnits: thf
+		LoadedCondition: loaded
+	Armament@AA:
+		Weapon: 90mmAA
+		Recoil: 128
+		RecoilRecovery: 38
+		LocalOffset: 720,0,80
+		MuzzleSequence: muzzle
+	AttackTurreted:
+		RequiresCondition: loaded
+	Turreted:
+		RequiresCondition: loaded
+	Targetable@IGNORE:
+		TargetTypes: NoAutoTarget
+		RequiresCondition: !loaded
+
+1TNK:
+	Huntable:
+		RequiresCondition: loaded
+	Mobile:
+		RequiresCondition: !being-captured && loaded
+	Cargo:
+		Types: Pilot
+		MaxWeight: 1
+		PipCount: 1
+		InitialUnits: thf
+		LoadedCondition: loaded
+	AttackTurreted:
+		RequiresCondition: loaded
+	Turreted:
+		RequiresCondition: loaded
+	Targetable@IGNORE:
+		TargetTypes: NoAutoTarget
+		RequiresCondition: !loaded
+
+ARTY:
+	Huntable:
+		RequiresCondition: loaded
+	Mobile:
+		RequiresCondition: !being-captured && loaded
+	Cargo:
+		Types: Pilot
+		MaxWeight: 1
+		PipCount: 1
+		InitialUnits: thf
+		LoadedCondition: loaded
+	AttackFrontal:
+		RequiresCondition: loaded
+	Targetable@IGNORE:
+		TargetTypes: NoAutoTarget
+		RequiresCondition: !loaded
+
+OILB:
+	-Targetable:
+	-Selectable:
+	Interactable:
+	Explodes:
+		Weapon: Atomic
+	-TooltipDescription@ally:
+	-TooltipDescription@other:
+
```

Comment:
Extra space.

Reply:
We can discuss/implement these changes. I'm working on the helicopter logic now.